src/pages/users/index.tsx

```diff
@@ -0,0 +1,42 @@
+import './users-list.css';
+import { useGetUsers } from "../../hooks/useGetUsers";
+import { UserDto } from '../../types/user';
+import { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import { checkLoginStatus } from '../../utils/auth';
+
+export default function UsersList() {
+    if (!checkLoginStatus()) Navigate({ to: '/login' })
+
+    // Define the offset of the pagination
+    const [offset, setOffset] = useState(0);
+    const { data, loading } = useGetUsers({ offset });
```

Comment:
se aqui é passado apenas o offset, como você garante que uma página tem 10 usuários? e se esse valor padrão for modificado no back?

Reply:
Não cogitei essa possibilidade, vou alterar isso!

Reply:
Já alterei